artifactory/commands/golang/archive.go

```diff
@@ -152,7 +143,31 @@ func archiveProject(writer io.Writer, dir, mod, version string) error {
 		return err
 	}
 
-	return Create(writer, m, files)
+	return gozip.Create(writer, m, files)
+}
+
+func getAbsolutePaths(exclusionPatterns []string) ([]string, error) {
+	var absolutedPaths []string
+	for _, singleExclusion := range exclusionPatterns {
+		singleExclusion, err := filepath.Abs(singleExclusion)
+		if err != nil {
+			return nil, err
+		}
+		absolutedPaths = append(absolutedPaths, singleExclusion)
+	}
+	return absolutedPaths, nil
+}
+
+func isPathExcluded(path string, excludePathPattern string) (excluded bool, err error) {
```

Comment:
The code of this function does not match the input it receives. The function receives `excludePathPattern`, which is a tring fo multiple patterns, but treats it as a single pattern. I believe I have already pointed out this issue before. Before fixing the issue, please create a test that detects the above issue. See that the test fails, and only then fix the issue.